redfish-core/lib/log_services.hpp

```diff
@@ -1112,17 +1251,71 @@ inline void createDump(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                 "DiagnosticDataType & OEMDiagnosticDataType");
             return;
         }
-        if ((*oemDiagnosticDataType != "System") ||
+
+        if ((*oemDiagnosticDataType != "System") &&
             (*diagnosticDataType != "OEM"))
         {
-            BMCWEB_LOG_ERROR("Wrong parameter values passed");
+            BMCWEB_LOG_WARNING("Wrong parameter values passed");
```

Comment:
This should not be an internal error.. This is a user error

Reply:
Warning is correct here but the internalError is not

Reply:
done